src/test/java/net/masterthought/cucumber/json/StepTest.java

```diff
@@ -114,6 +114,36 @@ public void getEmbeddings_ReturnsEmbeddings() {
         assertThat(embeddings[0].getMimeType()).isEqualTo("application/json");
     }
 
+    @Test
+    public void hasEmbeddings() {
```

Comment:
make method more descriptive so this is clear what does it test - check with other methods